R CODE FINAL Alexander Shemetev

```diff
@@ -0,0 +1,791 @@
+  difference9 = abs(sensivity9-specifity9)
+  difference10 = abs(sensivity10-specifity10)
+  difference11 = abs(sensivity11-specifity11)
+  difference12 = abs(sensivity12-specifity12)
+  difference13 = abs(sensivity13-specifity13)
+  difference14 = abs(sensivity14-specifity14)
+  difference15 = abs(sensivity15-specifity15)
+  difference16 = abs(sensivity16-specifity16)
+  difference17 = abs(sensivity17-specifity17)
+  difference18 = abs(sensivity18-specifity18)
+  difference19 = abs(sensivity19-specifity19)
+  
+  difa=c(difference1,difference2,difference3,difference4,difference5,difference6,difference7,difference8,difference9,difference10,difference11,difference12,difference13,difference14,difference15,difference16,difference17,difference18,difference19)
+  mindifa = min(difa)
+  sumsen=c(sensivity1,sensivity2,sensivity3,sensivity4,sensivity5,sensivity6,sensivity7,sensivity8,sensivity9,sensivity10,sensivity11,sensivity12,sensivity13,sensivity14,sensivity15,sensivity16,sensivity17,sensivity18,sensivity19)
+  sumspec=c(specifity1,specifity2,specifity3,specifity4,specifity5,specifity6,specifity7,specifity8,specifity9,specifity10,specifity11,specifity12,specifity13,specifity14,specifity15,specifity16,specifity17,specifity18,specifity19)
+  acurainfo = c(0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4, 0.45, 0.5, 0.55, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9, 0.95)
+  
+  
+  
+  data.frame(acurainfo, difa, mindifa, sumsen, sumspec)
+  
+  
+}
+var_logit_sens_score(bankloans_new$income)
+# SUCCESS
+# Explanations to the function: the function exactly follows the provided formula in the set of this part
+# and it estimates what was asked to estimate: which percentile (acurainfo) corresponds with which difference (difa); it shows the minimum difference (mindifa - in our example for this column corresponds with 0.55 accuracy); sensivity (sumsen column); specifity (sumspec column)  
+
+# 5) Are the estimated thresholds different between the two models?
+# For example, lets take income column in the bankloans_new database
+var_logit(bankloans_existing$income)
+# Model estimated 95 percentile as the highest accurance 
+var_logit_sens_score(bankloans_existing$income)
+# Model prescribed 55 percentali as the highest accurance
+
+
+# 6) Predict the default probabilities for the bankloans_missing ( model.predict). 
+#Obtain the probabilities of default. 
+#    Convert them to booleans (1 or 0) based on either threshold estimated above. 
+# bankloans_missing IS THE FIRST TIME MENTIONED IN THE TASK - what is this? 
+
+bankloans_missing = bankloans_new
+logit <- glm(bankloans_existing$default ~ bankloans_existing$age + bankloans_existing$ed + bankloans_existing$employ + bankloans_existing$address + bankloans_existing$income + bankloans_existing$debtinc + bankloans_existing$creddebt + bankloans_existing$othdebt,family="binomial")
+summary(logit)
+pred <- predict(logit,bankloans_missing=data)
+pred
+probs <- exp(pred)/(1+exp(pred))
```

Comment:
no need to apply sigmoid transform as the preditctions are already probabilities.